container-search/src/main/resources/configdefinitions/container.search.schema-info.def

```diff
@@ -0,0 +1,25 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+namespace=container.search
+
+## The name of this schema
+schema[].name string
+
+## The id of the summary class
+# schema[].summaryclass[].id int
+## The name of the summary class
+# schema[].summaryclass[].name string
+## The name of a field in the summary class
+# schema[].summaryclass[].fields[].name string
+## The type of a field in the summary class
+# schema[].summaryclass[].fields[].type string
+## Whether this field is a dynamic snippet
+# schema[].summaryclass[].fields[].dynamic bool default=false
+
+## Information about rank profiles
+schema[].rankprofile[].name string
+schema[].rankprofile[].hasSummaryFeatures bool default=true
+schema[].rankprofile[].hasRankFeatures bool default=true
```

Comment:
i can't find any place where hasRankFeatures is actually used.

Reply:
This was easier to keep. It will be used when I move away from documentdb-info.